acceptance/steps/accessibility-statement.test.js

```diff
@@ -0,0 +1,41 @@
+const JestCucumber = require('jest-cucumber')
+const WebDriver = require('selenium-webdriver');
+const firefox = require('selenium-webdriver/firefox');
+
+const screen = {
+  width: 1024,
+  height: 768
+};
+
+const feature = JestCucumber.loadFeature('features/accessibility-statement.feature');
+
+JestCucumber.defineFeature(feature, test => {
+  test('Accessibility statement page contains expected heading', ({ given, when, then }) => {
+
+    let url = "";
+    let driver;
+
+    given('I am a Cyber Capable Person interested in accessibility', () => {
+      driver = new WebDriver.Builder()
+        .withCapabilities(WebDriver.Capabilities.firefox())
+        .setFirefoxOptions(new firefox.Options()
+          .headless()
+          .windowSize(screen)
+        )
+        .build();
+    });
+
+    when('I visit the Cyber Health Framework site Accessibility statement page', async () => {
+      const urlStub = 'accessibility-statement'
+      url = `${process.env.FRONTEND_PROTO}://${process.env.FRONTEND_HOST}:${process.env.FRONTEND_PORT}/${urlStub}`
```

Comment:
Could we pull this out into something that we can re-use? Perhaps one for another ticket?

Reply:
I've done _something_ to this effect that can probably be easiest viewed at the following diff: https://github.com/wearesnook/cyber-health-frontend/compare/26a125e..b378d23

I've only re-factored the one acceptance test ATM as I'm not convinced it's worth keeping or not! Curious to hear your thoughts :)

Reply:
OK'd with Joel on Friday - I'll refactor the others like this today!